Features/Context/SubContext/CommonActions.php

```diff
@@ -12,6 +12,91 @@
 trait CommonActions
 {
     /**
+     * @Given I clicked on/at (the) :link link
+     * @When  I click on/at (the) :link link
+     *
+     * Click a link with text ':link'
+     */
+    public function iClickAtLink($link)
```

Comment:
Could we please take advantage of this PR to remove this  "atLink" ? You don't click at something, but "on" something. We can keep the alternative in the sentences.

This is also valid for other methods in the same file.